server/gabble/chat/health_events.go

```diff
@@ -11,8 +10,8 @@ const (
 )
 
 // InitHealtEvents Basic health events
-func InitHealtEvents(server *socketio.Server) {
-	server.OnEvent(config.PATH, PingEvent, func(so socketio.Conn, msg string) {
+func InitHealtEvents(server *SocketIOServer) {
+	server.Socket.OnEvent(server.DefaultSocketPath, PingEvent, func(so socketio.Conn, msg string) {
```

Comment:
Read `DefaultSocketPath` from the `SocketIOServer`, this helps to have the `DefaultSocketPath` as part of the Server instance instead of using `/` for every instance.

This value is also customisable from the `.env` file